docs/kyma/docs/017-details-local-installation-scripts.md

```diff
@@ -3,21 +3,7 @@ title: Local installation scripts
 type: Details
```

Comment:
with this Pr we should not put type: Installation to all docs related to installation, and they do not have to follow Getting started flow